Translation corrections (#552)
07th-mod/higurashi-console-arcs: Update/koto_002.txt

//r『彼女』はその一つを制御するために作られたものだが、現在は私の手で改良を施し、鬼ヶ淵村一帯を中心に『同化』していった同朋たちの動向管理も担わせている。
	OutputLine(NULL, "『彼女』はその一つを制御するために作られたものだが、現在は私の手で改良を施し、鬼ヶ淵村一帯を中心に『同化』していった同朋たちの動向管理も担わせている。",
			NULL, "\"She\" was built to control one of those, but I'd made my own improvements to her, and now she was in charge of governing the actions of me and my brethren who had \"assimilated\" around the Onigafuchi Village area.", Line_Normal);
	ClearMessage();

//rまた、姿こそ持たないが…膨大な知識と記録を所有する『彼女』は私とのやり取りの中、生物のような感情や人格を疑似的に備えるようになった。
	OutputLine(NULL, "また、姿こそ持たないが…膨大な知識と記録を所有する『彼女』は私とのやり取りの中、生物のような感情や人格を疑似的に備えるようになった。",

> 			NULL, "And though she had no form... \"she\" learned to simulate creature-like feelings and character through exchanges with me based on a huge amount of knowledge and records,.", Line_Normal);

Reviewer: ```suggestion
			NULL, "And though she had no form... \"she\" learned to simulate creature-like feelings and character through exchanges with me, based on a huge amount of knowledge and records.", Line_Normal);
```

//rリューン＝オークの律義な指摘にとぼけてみたが、…実際手入れが面倒臭くて怠っていたのが一番の理由なので、それ以上の言い訳を諦めた。
	OutputLine(NULL, "リューン＝オークの律義な指摘にとぼけてみたが、…実際手入れが面倒臭くて怠っていたのが一番の理由なので、それ以上の言い訳を諦めた。",

> 			NULL, "I tried playing dumb at Ryuun-Ohc's honest reminder... but I gave up on further excuses since the biggest reason was that I slacked off since actually tending the blade was a pain in the ass.", Line_Normal);

Reviewer: ```suggestion
			NULL, "I tried playing dumb at Ryuun-Ohc's honest reminder... but I gave up on further excuses since the biggest reason was that I slacked off, since actually tending to the blade was a pain in the ass.", Line_Normal);
```

			NULL, "\"Come to think of it, Jedha...", Line_WaitForInput);
	PlaySE(4, "ps3/s16/00/ds43050050", 256, 64);
	OutputLine(NULL, "今回大陸には他のジェダと一緒に赴かれたと聞いておりましたが、それは以前お会いしたシーツェン＝ジェダですか？』",

> 			NULL, "I heard you worked with other Jedha on the mainland this time. Was it Shezhen Jedha, who we had met priorly?\"", Line_Normal);

Reviewer: ```suggestion
			NULL, "I heard you worked with other Jedha on the mainland this time. Was it Shezhen Jedha, who we met previously?\"", Line_Normal);
```

//羽入rvS16/45/DS43070188.「あはは、異なことを言う。kvS16/45/DS43070189.『bグリフィス.<混血>』を討ち果たすがために生きる、我らジェダを支援する役目のお前が、私に使命を忘れて休め、とはな。kvS16/45/DS43070190.…どういう風の吹きまわしだ？」
	PlaySE(4, "ps3/s16/45/ds43070188", 256, 64);
	OutputLine(NULL, "「あはは、異なことを言う。",
			NULL, "\"Ahaha, what an idea.", Line_WaitForInput);
	PlaySE(4, "ps3/s16/45/ds43070189", 256, 64);
	OutputLine(NULL, "『混血(グリフィス)』を討ち果たすがために生きる、我らジェダを支援する役目のお前が、私に使命を忘れて休め、とはな。",

> 			NULL, " You, who exists to support us Jedha who live to exterminate the Grifys, suggesting I forget my mission and get some rest?", Line_WaitForInput);

Reviewer: ```suggestion
			NULL, " You, who exist to support us Jedha who live to exterminate the Grifys, suggesting I forget my mission and get some rest?", Line_WaitForInput);
```
